Mathlib/LinearAlgebra/BilinearMap.lean

```diff
@@ -383,6 +383,32 @@ theorem compr₂_apply (f : M →ₗ[R] Nₗ →ₗ[R] Pₗ) (g : Pₗ →ₗ[R]
     f.compr₂ g m n = g (f m n) := rfl
 #align linear_map.compr₂_apply LinearMap.compr₂_apply
 
+/-- Apply linear equivalences on the arguments of a bilinear map. -/
+def congr (e₁ : Mₗ ≃ₗ[R] Nₗ) (e₂ : Pₗ ≃ₗ[R] Qₗ) :
```

Comment:
This should probably be called
```suggestion
def congrl₁₂ (e₁ : Mₗ ≃ₗ[R] Nₗ) (e₂ : Pₗ ≃ₗ[R] Qₗ) :
```

Reply:
I've made this change in #8255 and merged that branch into this.